thoth/metrics_exporter/jobs/solver.py

```diff
@@ -60,43 +60,81 @@ def get_unsolved_python_packages_count(cls) -> None:
+                python_version=solver_info["python_version"],
+            )
+            metrics.graphdb_total_number_solved_python_packages.labels(solver_name).set(count_solved)
+            _LOGGER.debug("graphdb_total_number_solved_python_packages(%r)=%r", solver_name, count_solved)
+
+    @classmethod
+    @register_metric_job
+    def get_python_packages_solver_error_count_per_solver(cls) -> None:
+        """Get number of python packages with solver error True and how many are unparsable or unsolvable per solver."""
+        for solver_name in cls._OPENSHIFT.get_solver_names():
+            solver_info = cls.graph().parse_python_solver_name(solver_name)
+            total_python_packages_solved = cls.graph().get_solved_python_packages_count_all(
+                os_name=solver_info["os_name"],
+                os_version=solver_info["os_version"],
+                python_version=solver_info["python_version"],
+            )
+
+            total_python_packages_solver_error = cls.graph().get_error_solved_python_package_versions_count_all(
+                os_name=solver_info["os_name"],
+                os_version=solver_info["os_version"],
+                python_version=solver_info["python_version"],
+            )
+            total_python_packages_solver_error_unparseable = cls.graph().get_error_solved_python_package_versions_count_all(
+                unparseable=True,
+                os_name=solver_info["os_name"],
+                os_version=solver_info["os_version"],
+                python_version=solver_info["python_version"],
+            )
+            total_python_packages_solver_error_unsolvable = cls.graph().get_error_solved_python_package_versions_count_all(
+                unsolvable=True,
+                os_name=solver_info["os_name"],
+                os_version=solver_info["os_version"],
+                python_version=solver_info["python_version"],
+            )
+
+            total_python_packages_solved_with_no_error = total_python_packages_solved - total_python_packages_solver_error
+
+            metrics.graphdb_total_python_packages_solved_with_no_error.labels(solver_name).set(
+                total_python_packages_solved_with_no_error
+            )
+            metrics.graphdb_total_python_packages_with_solver_error_unparseable.labels(solver_name).set(
+                total_python_packages_solver_error_unparseable
+            )
+            metrics.graphdb_total_python_packages_with_solver_error_unsolvable.labels(solver_name).set(
+                total_python_packages_solver_error_unsolvable
+            )
+            metrics.graphdb_total_python_packages_with_solver_error.labels(solver_name).set(
+                total_python_packages_solver_error
+            )
+
+            _LOGGER.debug(
+                "graphdb_total_python_packages_solved_with_no_error=%r",
+                solver_name,
+                total_python_packages_solved_with_no_error
```

Comment:
You pass 2 arguments to logging, but only one will be used - this will cause formatting exceptions on logger side. Here and bellow as well.